### PR TITLE
calcurse: add pt-br translation

### PR DESCRIPTION
**Pull request by fabiobianchiwebdev**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages.pt-BR/linux/calcurse.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#calcurse
+
+>Um calendário e agenda baseada em texto para a linha de comando.
```
Comment (waldyrious) [inline pages.pt-BR/linux/calcurse.md]:
```suggestion
> Um calendário e agenda baseados em texto para a linha de comando.
```

File: pages.pt-BR/linux/calcurse.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Inicie o calcurse em modo interativo:
+
+`calcurse`
+
+- Mostra os agendamentos e eventos para o presente dia:
+
+`calcurse -a` or `calcurse --appointment`
+
+- Apaga todos os ítens gravados localmente no calcurse e importa os objetos gravados remotamente:
```
Comment (waldyrious) [inline pages.pt-BR/linux/calcurse.md]: Até este ponto não foi mencionado nada que possa indicar a que o "remotamente" se refere.

Seria possível acrescentar um exemplo anterior a este, a mostrar como essa ligação se faz, ou a listar as ligações existentes (como por exemplo `git remote -v` faz no caso do git)?

Alternativamente, uma nota na descrição principal poderia servir também, caso um exemplo de comando seja impraticável.

Comment (andrik) [reply pages.pt-BR/linux/calcurse.md]: Olá @fabiobianchiwebdev

Verifiquei na documentação que o calcurse-caldav é uma extensão ao calcurse, então sugiro separarmos em dois comandos. Dessa forma, a própria descrição do calcurse-caldav esclareceria a utilização do termo "remotamente".

Grato pela contribuição.

**Comment by sbrl** [comment]
Note that you also need to sign the CLA as  @CLAassistant mentions in their comment. Just click the link :-)

If it doesn't work, then you've probably used the incorrect email address in your commits. It has to be an email address that's attached to your GitHub account.

File: pages.pt-BR/linux/calcurse.md (hunk 2 above)
Comment (fabiobianchiwebdev) [reply pages.pt-BR/linux/calcurse.md]: Grato pelas sugestões @andrik . Vou fazer novos pull-request com as mudanças necessárias.

**Comment by waldyrious** [comment]
> If it doesn't work, then you've probably used the incorrect email address in your commits. It has to be an email address that's attached to your GitHub account.

Note that you can add multiple email addresses to you account, and choose which of them (if any) is made visible in your profile.

**Comment by fabiobianchiwebdev** [comment]
I need to sign the CLA every time I do a pull request?

**Comment by schneiderl** [comment]
@fabiobianchiwebdev no. you need to sign the CLA only once for each user/email you commit with. Are you creating a commit with a different email then you did previously?

**Comment by schneiderl** [comment]
Thanks a lot for the translation, 
If you could change the PR title to follow what is usually done on tldr. Something like "calcurse.md: add pt br translation" 
Is not a big deal since we can change the PR title while merging it but it would be good for your following PRs to follow that style. 

Once again thank you.

**Comment by waldyrious** [comment]
@fabiobianchiwebdev, most likely what happened here regarding the email is that your git author email, as configured in the terminal (see it with `git config user.email`) is not connected to your github account, as shown in https://github.com/settings/emails.

Please add your email there, which will probably solve the issue with CLA-assistant :)

**Comment by fabiobianchiwebdev** [comment]
As far as I know, I'm using the same e-mail. But because it's my first time working files on github I could have made a mistake.

**Comment by waldyrious** [comment]
> I just checked and the e-mail on settings is the same I'm using.

Then maybe you made the first few commits in an environment with a git author email that is not listed in github.com/settings/emails. This is what I see if I check out this PR's branch:

```sh
❯ git lg --pretty=full -s
* commit e242ff88 (HEAD -> calcurse-pt_BR)
| Author: Fabio Bianchi <13044134+fabiobianchiwebdev@users.noreply.github.com>
| Commit: GitHub <noreply@github.com>
|
|     Update pages.pt-BR/linux/calcurse.md
|
|     Co-Authored-By: Waldir Pimenta <waldyrious@gmail.com>
|
* commit c272b7c6
| Author: Fabio Bianchi <con...@gmail.com>
| Commit: Fabio Bianchi <con...@gmail.com>
|
|     calcurse: correção do tempo verbal
|
* commit 99b184f0
| Author: Fabio Bianchi <con...@gmail.com>
| Commit: Fabio Bianchi <con...@gmail.com>
|
|     calcurse: add pt-BR translation
|
* commit 945b9d87
| Author: Fabio Bianchi <con...@gmail.com>
| Commit: Fabio Bianchi <con...@gmail.com>
|
|     calcurse: add page
```

Note how the commits you authored online (by accepting review suggestions) are associated with the email `<13044134+fabiobianchiwebdev@users.noreply.github.com>`, while those that authored on your machine show a `gmail.com` email address. That's the one you should add to your GitHub account for the commits to be recognized as yours.

**Comment by fabiobianchiwebdev** [comment]
You're right @waldyrious, I used a different e-mail by mistake. I'm just starting using git with the command line... 
I will review the procedures nest time. Thanks for helping me.

**Comment by waldyrious** [comment]
> What I have to do now for fix that?

As I mentioned above, if you just add the gmail address listed in the commits to your GitHub account at https://github.com/settings/emails, GitHub (and CLA-assistant) will retroactively recognize the existing commits made with it and associate them to your account.

**Comment by waldyrious** [comment]
Update: I've collapsed the comments related to the CLA issue, as that now seems to be resolved.

**Comment by waldyrious** [comment]
Rebased the branch on top of the latest master, and applied remaining review comments. Splitting the pages (this one and the English one) in two can be done in a separate PR.

**Pull request merged**